gophpserialize.go

```diff
@@ -214,7 +214,13 @@ func (s *Serializer) readValue() interface{} {
 		s.move()
 		return nil
 	}
-	panic("Unknown objType: " + string(objType) + "\n" + string(s.raw))
+
+	panic(fmt.Sprintf(
+		"unknown object type: `%s` in ...%s... in position: %v",
```

Comment:
sry for nitpicking again, should it be `at position`?

Reply:
Sounds better 👍  addressed in https://github.com/zalora/gophpserialize/pull/2/commits/7884dbb422dbdd3ce209578c24ef78646b0f7818